parm/config/config.resources.emc.dyn

```diff
@@ -192,15 +195,16 @@ elif [ $step = "fcst" ]; then
     export npe_node_fcst=$(echo "$npe_node_max / $nth_fcst" | bc)
     export npe_node_fcst_gfs=$(echo "$npe_node_max / $nth_fcst_gfs" | bc)
     if [[ "$machine" == "WCOSS2" ]]; then
+       export wtime_fcst_gfs="02:30:00"
        export npe_node_fcst=32
-       export npe_node_fcst_gfs=42
+       export npe_node_fcst_gfs=24
```

Comment:
How do the values on L200 and L199 differ from the calculations on L195 and L196?

Reply:
L195 = 128/3 = 42.6, which rounds up to 43 in the xml
L196 = 128/5 = 25.6, which rounds up to 26 in the xml

Neither value lays nicely across the WCOSS2 nodes.

George V and I did a lot of testing to get the values on L199 and L200. We want users using these values for C768 on WCOSS2. May also want them for lower resolutions too but haven't fully vetted those resolutions on WCOSS2 yet.

This reminded me I forgot to adjust the C768 block in `config.fv3.emc.dyn` with the tested values for `layout_x_gfs` and `WRTTASK_PER_GROUP_GFS` from `config.fv3.nco.static`. Please see that change in this PR now too.

Reply:
Ok. So these values are specific to C768.
May be then please add a note to that effect.

Reply:
Done. I added a check for CASE=768 to make sure those values aren't used outside of high res on WCOSS2.